ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV1.java

```diff
@@ -39,4 +41,14 @@ public EngineNewPayloadV1(
   public String getName() {
     return RpcMethod.ENGINE_NEW_PAYLOAD_V1.getMethodName();
   }
+
+  @Override
+  protected boolean requireTerminalPoWBlockValidation() {
```

Comment:
Do these need to be methods? Could just configure Abstract class by passing the in these values. What would be more readable though?